src/software/world/robot.h

```diff
@@ -50,6 +50,9 @@ class Robot
 
     /**
      * Creates a new robot based on the TbotsProto::Robot protobuf representation
+     * NOTE: TbotsProto::Robot does not store the robot constants and the original robot
+     * constants is lost in the conversion, as a result, the 2021 robot constants are used
+     * as default.
```

Comment:
Hmm, we could probably use an enum if we want to add it to the Robot proto (Adding RobotConstants_t for every robot might be unnecessarily large). The current approach for HRVO is to get the Robot Constants directly from Thunderloop or ErForceSim, so it wouldn't use the Robot Constant in the proto anyways. But if we want to make the proto future proof, we could add it. What do you think?

Reply:
RobotConstants should be a part of the robot software and reported back to us by each robot, perhaps on start up

Reply:
So do you think it's okay to keep it as is? Since HRVO running on the robots uses the RobotConstants from Thunderloop, so I guess it could be configured there if we want that functionality.

Reply:
I think it's a good note to have, it's something figure out for fleet management